### ISSUE TITLE
package not install on livewire 11

### ISSUE THREAD

**Issue by lokiwich** [tzsk/sms#1384]
I tried to install the package on Laravel 11, but it gives this error when installing:

```
composer require tzsk/sms
./composer.json has been updated
Running composer update tzsk/sms
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - tzsk/sms[1.0.0, ..., 1.2.1] require illuminate/support ~5.2 -> found illuminate/support[v5.2.0, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[1.3.0, 2.0.0, ..., 2.0.2] require illuminate/support ~5.1 -> found illuminate/support[v5.1.1, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[3.0.0, ..., 3.0.2] require illuminate/support ~5.7|~5.8|^6.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[4.0.0, ..., 4.0.1] require illuminate/support ~5.7|~5.8|^6.0|^7.0 -> found illuminate/support[v5.7.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[5.0.0, ..., 5.0.2] require php ^7.4 -> your php version (8.2.0) does not satisfy that requirement.
    - tzsk/sms[5.1.0, ..., 5.4.1] require illuminate/http ^7.0|^8.0 -> found illuminate/http[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[6.0.0, ..., 6.0.1] require illuminate/http ^8.0|^9.0 -> found illuminate/http[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - tzsk/sms[7.0.0, ..., 7.0.1] require illuminate/http ^8.0|^9.0|^10.0 -> found illuminate/http[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.3] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires tzsk/sms * -> satisfiable by tzsk/sms[1.0.0, ..., 1.3.0, 2.0.0, 2.0.1, 2.0.2, 3.0.0, 3.0.1, 3.0.2, 4.0.0, 4.0.1, 5.0.0, ..., 5.4.1, 6.0.0, 6.0.1, 7.0.0, 7.0.1].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require tzsk/sms:*" to figure out if any version is installable, or "composer require tzsk/sms:^2.1" if you know which you need.
```

**Comment by tzsk** [comment]
Laravel 11 compatibility needs to be updated. Will be released this week.

**Comment by lokiwich** [comment]
Thank you

**Comment by tzsk** [comment]
Released `v8.0.0` with Laravel 11 Compatibility.
